src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -233,7 +233,26 @@ public static DataMarshaler OutputMarshaler
                                 }
                             }
 
-                            return outputMarshaler.Marshal(pyObj.AsManagedObject(typeof(object)));
+                            var unmarshalled = pyObj.AsManagedObject(typeof(object));
```

Comment:
Can we just have the using here:
`using (var unmarshalled = pyObj.AsManagedObject(typeof(object)))`
and remove the one in line 239?

Reply:
No, we can't. The problem is that `AsManagedObject` returns `object` so that code wouldn't compile. We want to dispose it if what we get back is a `PyObject`, but that's not always the case.